Dockerfile

```diff
@@ -211,4 +225,4 @@ ENTRYPOINT ["/sbin/docker-entrypoint.sh"]
 
 EXPOSE 8000
 
-CMD ["/usr/local/bin/supervisord", "-c", "/etc/supervisord.conf"]
+CMD ["/usr/local/bin/paperless_cmd.sh"]
```

Comment:
Nit: I can't say I love this naming for the file, it doesn't feel descriptive to what it does

Reply:
It's the docker `cmd` to run paperless - I wasn't feeling particularly inspired and will heartily accept recommendations.